.studio/notifications-service

```diff
@@ -120,7 +120,7 @@ function notifications_update_component() {
   pushd /src/components/notifications-service/server >/dev/null;
     mix local.hex --force
     mix local.rebar --force
-    MIX_ENV=habitat mix do deps.get, compile
+    MIX_ENV=habitat mix "do" deps.get, compile
```

Comment:
Hm? OK.

Reply:
`do` is a shell keyword so things get confused. 🤷‍♂ I didn't look too far into it.

Reply:
Yeah suspected something like that. 🚤